packages/sdks-tests/src/tests/blocks.spec.ts

```diff
@@ -287,4 +289,120 @@ test.describe('Blocks', () => {
       });
     }
   });
+
+  test.describe('Test ApiVersion', () => {
+    testOnlyOldReact('apiVersion in old react is no set', async ({ page }) => {
+      await page.goto('/api-version-default');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v1\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
+
+    testOnlyOldReact('apiVersion in old react is set to v1', async ({ page }) => {
+      await page.goto('/api-version-v1');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v1\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
+
+    testOnlyOldReact('apiVersion in old react is set to v3', async ({ page }) => {
+      await page.goto('/api-version-v3');
+
+      let x = 0;
+      await page.route(
+        /.*cdn\.builder\.io\/api\/v3\/content\/symbol.*/,
+        route => {
+          x++;
+          return route.fulfill({
+            status: 200,
+            body: x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT,
+          });
+        },
+        { times: 2 }
+      );
+
+      await testSymbols(page);
+    });
```

Comment:
for the `v3` test, we don't need 2 separate tests for oldReact and the new SDKs. You can just run `test()` normally and have it work on all of them at once

```suggestion
```